src/lib/abacus/rest.ts

```diff
@@ -56,13 +56,8 @@ class AbacusRest implements AbacusRestProtocol {
       .then(async (response) => {
         const data = await response.text();
 
-        if (response.ok) {
-          callback(data, response.status);
-        } else {
-          // response not OK, call callback with null data and the status, this includes 400/500 status codes
-          callback(null, response.status);
-        }
```

Comment:
Are we sure this won't affect handling of any other rest request?

Reply:
I think it should be ok, the filtering is not needed. It looks like IOS also does not have logic filtering based on status code: https://github.com/dydxprotocol/v4-native-ios/blob/b5d65b38e693801f5ba0acfde1d6bb4c8ab8ba9c/dydx/dydxStateManager/dydxStateManager/Protocol%20Implementations/AbacusRestImp.swift#L67C24-L67C28

so if abacus is working well for IOS, work.